lib/datadog/tracing/contrib/grape/endpoint.rb

```diff
@@ -61,6 +62,15 @@ def endpoint_start_process(_name, _start, _finish, _id, payload)
               span.set_tag(Tracing::Metadata::Ext::TAG_COMPONENT, Ext::TAG_COMPONENT)
               span.set_tag(Tracing::Metadata::Ext::TAG_OPERATION, Ext::TAG_OPERATION_ENDPOINT_RUN)
 
+              if (grape_route = env['grape.routing_args'] && env['grape.routing_args'][:route_info])
+                trace.set_tag(
+                  Tracing::Metadata::Ext::HTTP::TAG_ROUTE,
+                  grape_route.path&.gsub(/\(\.{1}:?\w+\)\z/, '')
```

Comment:
`{1}` is redundant, no?

Reply:
yes, thanks!

Reply:
fixed [3207b16](https://github.com/DataDog/dd-trace-rb/pull/3849/commits/3207b16ea39d9a5b7af0d7a1ae392058d2c39656)